lsp/odoo_ls.lua

```diff
@@ -0,0 +1,33 @@
+-- This file provides the default configuration for the 'odoo_ls' server.
+
+local odoo_ls_locations = {vim.fn.expand('$HOME/.local/share/nvim/odoo/odoo_ls_server')}
+
+local executable = ''
+if vim.fn.executable('odoo_ls_server') then
+    executable = 'odoo_ls_server'
+end
+
+
+for _, location in ipairs(odoo_ls_locations) do
+    local file = io.open(location, "r")
+    if file ~= nil then
+        io.close(file)
+        executable = location
+    end
+end
```

Comment:
```suggestion
local odoo_ls_locations = {
    vim.fn.expand('$HOME/.local/share/nvim/odoo/odoo_ls_server')
    "odoo_ls_server",
}

for _, location in ipairs(odoo_ls_locations) do
    if vim.fn.executable('location') then
        executable = location
        break
    end
end
```
It should be simpler to test only the execution then return the config